main/inc/lib/extra_field_option.lib.php

```diff
@@ -244,7 +244,7 @@ public function save($params, $show_query = false)
                 foreach ($list as $option) {
                     $option_info = self::get_field_option_by_field_and_option($field_id, $option);
                     // Use URLify only for new items
-                    $optionValue = URLify::filter($option);
+                    $optionValue = $option;
```

Comment:
The URLify class doesn't exists; I've replaced with the replace_dangerous_char function